src/main/java/com/cmsc495/spacesim/model/Resources.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package com.cmsc495.spacesim.model;
+
+/**
+ *
+ * @author NoorQasim
+ */
+public class Resources {
+    
+   private String[] resource = {"Food", "Water", "Gasoline", "Coal", "Medical Supplies", "Builidng Materials"};

```

Comment:
You should do the same as I mentioned for the `allPossibleSkills` array here for these possible resource values.